### DISCUSSION TITLE
Boxxer high round

### DISCUSSION THREAD

**Discussion by mmkkiinn** [nzp-team/nzportable#955] [Records and Speedruns]
![Screenshot_2024-10-30-10-37-22-79_2f85358b2198d26f8aca533d68bee793](https://github.com/user-attachments/assets/032d5c85-3d71-425e-a7b3-708c31cda1e6)
The map very hard who can break my round

**Comment by Taswp** [comment]
![image](https://github.com/user-attachments/assets/e495859c-2ff0-4ca6-84b9-c6d309a9931c)
I beat your record. As of now I am the current record holder for Boxxer

**Comment by Galvinh0slk4** [comment]
my high round of boxxer hahaha (I was in 2nd place)
![fte-20241130174531-0](https://github.com/user-attachments/assets/c19eed1c-a784-4e04-a25c-02fa27bc670f)

**Reply by Taswp** [reply]
I do not think that this map can be taken to 50. That being said, I think we can push it as close as possible.

**Reply by Taswp** [reply]
Also we could do No power challenges XD

**Comment by Lolinburgs** [comment]
![boxx round 30](https://github.com/user-attachments/assets/c4562010-9cc0-43cd-8223-3cebc62b33c6)

did i do a thing

**Comment by DGxShAdOwZz** [comment]
![Screenshot 2024-12-12 111430](https://github.com/user-attachments/assets/e7d4db0c-b9c3-4fa2-ac94-4b3aed25c409)

Ill take a 38 thats not too bad

**Reply by Lolinburgs** [reply]
how is that humanly possible

**Reply by Lolinburgs** [reply]
wdym NOT TOO BAD???

**Reply by Lolinburgs** [reply]
I WAS FIGHTING FOR MY LIFE ON 30!

**Reply by DGxShAdOwZz** [reply]
40 is 100% do able, i just lost focus xD

**Reply by Lolinburgs** [reply]
I DIDNT EVEN THINK 35 WAS POSSIBLE WHEN I PLAYED

**Comment by BrownNPC** [comment]
he got that far playing the psp version on a touch screen, respect

**Reply by mmkkiinn** [reply]
Thanks for that

**Reply by Taswp** [reply]
I mean he could if used save states but I ain't accusing. Also, psp zombies only has 18 zombies per horde making it easier than you think. Plus you can hook up a controller to the emulator so who knows if he actually did it all on touch screen. Plus 6878 fps? yeah no that was done on pc emulator lol As time goes on for this game these are things you guys are gunna have to think about

**Reply by Taswp** [reply]
Also the og poster still had the touch controls on screen so he ACTUALLY went to round 18 using nothing but the touchscreen. Or at least he has more proof. I use emulators on my phone and the onscreen controls go away when a controller is hooked up. And, no I'm not gonna believe that he got rid of them for the screen shot.

**Reply by mmkkiinn** [reply]
No i use psp emulator for my phone

**Reply by Taswp** [reply]
I know you did I was talking about the guy who made it to 38

**Reply by MotoLegacy** [reply]
Spelling out slurs backwards won't make you immune to moderation.

**Reply by Taswp** [reply]
That's not exactly why I spelled it backwards but I see your point, sorry.

**Reply by DGxShAdOwZz** [reply]
> Also the og poster still had the touch controls on screen so he ACTUALLY went to round 18 using nothing but the touchscreen. Or at least he has more proof. I use emulators on my phone and the onscreen controls go away when a controller is hooked up. And, no I'm not gonna believe that he got rid of them for the screen shot.

Dont wanna be that guy - But there are other clients, like a pc client hence the 6000fps

**Reply by Taswp** [reply]
i kind of clarified that in my first post. No matter, will be grinding round 50 any ways. If you or anyone else reaches first please record so I can figure out any other strats other than camping in corner on insta kills and training the perimeter of the map.

**Reply by Taswp** [reply]
> > Also the og poster still had the touch controls on screen so he ACTUALLY went to round 18 using nothing but the touchscreen. Or at least he has more proof. I use emulators on my phone and the onscreen controls go away when a controller is hooked up. And, no I'm not gonna believe that he got rid of them for the screen shot.
> 
> Dont wanna be that guy - But there are other clients, like a pc client hence the 6000fps

Plus your graphics look exactly like the psp version so idk.

**Reply by DGxShAdOwZz** [reply]
all versions look alike - its just quake

**Reply by Taswp** [reply]
Uh no they don't? the past few screenshots clearly shows that...

**Reply by Taswp** [reply]
either way you got 38. good stuff. get that first 50

**Comment by Taswp** [comment]
![image](https://github.com/user-attachments/assets/f17faef2-b20e-4ad4-835a-4da3842c313b)

Boom Round 40!
I know this map can be taken to 50 for sure now. I just gotta keep practicing.

**Reply by Taswp** [reply]
Will be posting gameplay soon. Once uploaded will post link. Will be posting full gameplay.

**Reply by Taswp** [reply]
https://www.youtube.com/watch?v=QGqjWtg2Efk

**Comment by Taswp** [comment]
![image](https://github.com/user-attachments/assets/362e3afd-2d7c-4a62-a648-b7093a170c60)
Current 2 player record

**Reply by Taswp** [reply]
map is way more chaotic on co-op lolol

**Reply by BrownNPC** [reply]
We got the same 21, but I think we could have gotten to at least 24.

We died cuz the both of us were leading our own trains, then we reached the wall at the same time and gave each other a hug while 2 trains of zombies ate us from both sides
![image](https://github.com/user-attachments/assets/703427c1-669f-40c2-9491-61701d164932)

**Comment by Kingjaxgaming420** [comment]
![image](https://github.com/user-attachments/assets/3b845eae-a35a-4ce0-954b-4415031447d0)
technically the wr is 48, but that was played on an older version, so here's the wr for current version

**Comment by Kingjaxgaming420** [comment]
![image](https://github.com/user-attachments/assets/c0843c18-d920-4713-92e2-8604268cb7d9)
To anyone who is gonna try to beat this, good luck.

**Reply by Kingjaxgaming420** [reply]
https://youtu.be/pH7_p0MZji0?si=PYoAgNEPNEt9eNpr

**Reply by architsasha45-afk** [reply]
omg
bro

**Reply by architsasha45-afk** [reply]
I will beat it

**Reply by Kingjaxgaming420** [reply]
Honestly, it's not that hard once you know how to train them properly, and I do think somewhere between 60-70 is possible

**Reply by IASB2024** [reply]
yeah right

**Reply by ProfessorZIRE** [reply]
I used to grind boxer but it was too easy, so high round shouldn't be easy to get wr

**Reply by ProfessorZIRE** [reply]
Also why do u have edge pinned

**Comment by squidfries** [comment]
<img width="685" height="850" alt="image" src="https://github.com/user-attachments/assets/14f26c46-77f2-481a-b4e3-26fd76816023" />
 
New wr we downed because we are stupid but I feel like round35+ is possible

**Reply by envictonyt-sys** [reply]
W man we cooked

**Reply by M0THRAK** [reply]
Round 27 is insane! Great job!

**Comment by squidfries** [comment]
<img width="719" height="881" alt="image-1" src="https://github.com/user-attachments/assets/ce8d1659-6b78-40d4-9bd1-2c9277e26952" />

**Reply by squidfries** [reply]
ROUND 30 HAD BEEN ACHIEVED

**Reply by envictonyt-sys** [reply]
WE COOKIN

**Reply by Kingjaxgaming420** [reply]
You guys know I have a 50 right?

**Reply by Kingjaxgaming420** [reply]
Wait I'm stupid this is 2p my bad

**Reply by squidfries** [reply]
Jax im coming for your record i have a round 44 and im coming for you bro..

**Reply by Kingjaxgaming420** [reply]
good luck, if you do beat it i might come back for it

**Comment by squidfries** [comment]
<img width="3060" height="4080" alt="20260717_162647" src="https://github.com/user-attachments/assets/f836e27d-52c5-4584-9bb5-fba8cb9cbeed" /> I WAS SI CLOS3 I K OW I CAN BEAT IT IF IM LESS STUPID

**Reply by squidfries** [reply]
I meant I was so close but I was too excited to type properly

**Reply by FADEDAGAIN** [reply]
round 49 on a school computer is not bad

**Comment by squidfries** [comment]
the cold streak of almost a 2 years with no wr ends soon

**Reply by Taswp** [reply]
I might come back to take it back

**Reply by squidfries** [reply]
I count on it

**Comment by nonMat** [comment]
First time trying to high round this map!
It was fun!


<img width="1920" height="1080" alt="immagine" src="https://github.com/user-attachments/assets/4d6eff45-316e-4a03-b2c6-40eb0ce7e3de" />
